### DISCUSSION TITLE
Delete only selected resources

### DISCUSSION THREAD

**Discussion by ramzsey** [usememos/memos#4200] [Q&A] [answered]
**Hello,**
It might have been asked before, but I could not find it.
Neither in the issues, nor in the discussions.

Is it possible, to delete only selected resources?
Not all of them

**Comment by RoccoSmit** [comment]
How are you able to delete resources?

**Reply by ramzsey** [reply]
Well, if i attach any file to any of my memos, it is directly available in the resources section.
![image](https://github.com/user-attachments/assets/cb1ac79d-276e-4f77-9a69-48ad2269b37d)
From there, it is possible to delete all attached resources at once.
But I cannot manually select some and delete them.

**Reply by RoccoSmit** [reply]
These unused resources are waiting for your memo to be saved so that they can be linked to the memo. Once linked you cant delete them.
Are you adding resources to a memo so that they are displayed under unused resources and then not saving your memos but keeping the resources for a different purpose?

**Reply by ramzsey** [reply]
Got it!
Thank you so much )

**Reply by ramzsey** [reply]
But the attached files are not shown for whatever reason.
Created a new memo, attached a pdf file, saved the memo, opened it in a new tab, but the file is not attached yet.

**Reply by RoccoSmit** [reply, marked as answer]
I'm not able to replicate this. PDFs show up as expected. 
Note, in a memo PDFs appear as a text filename that you need to click on for the PDF to open in a new tab

![image](https://github.com/user-attachments/assets/da8c3390-dace-4c5f-b41b-67e1b59cf046)

**Reply by ramzsey** [reply]
Thank you so much for your help!
